### PR TITLE
Sum up to eight

### PR DESCRIPTION
**Pull request by JorgePatrick**
Implementation to sum roman numbers up to eight (VIII)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,16 +5,7 @@ build/
...
-.idea/jarRepositories.xml
-.idea/compiler.xml
-.idea/libraries/
-*.iws
-*.iml
-*.ipr
-out/
-!**/src/main/**/out/
-!**/src/test/**/out/
+.idea
```
Comment (chicofaccin) [inline .gitignore:8]: 👍

File: src/main/java/com/jorgepatrick/NumberConverter.java [hunk 2]
```diff
@@ -37,9 +67,20 @@ public int parseRomanToArabic(final String romanNumber) {
 
         for (int i = 0; i < romanNumber.length(); i++) {
             if (romanNumber.toUpperCase().charAt(i) == I.charAt(0)) {
-                unit += 1;
+                int j = i + 1;
```
Comment (chicofaccin) [inline src/main/java/com/jorgepatrick/NumberConverter.java:73]: If you use more meaningful names for the variables, it would be easier to read the code.

File: src/main/java/com/jorgepatrick/NumberConverter.java [hunk 3]
```diff
@@ -1,34 +1,64 @@
...
+
+        return romanNumber;
+    }
 
-        if (arabicNumberSum == 4) {
-            return I + V;
+    private String setRomanDigit(int arabicDigit, Digits digit) {
+        final String[] RomanOnes = {"I", "X", "C", "M"};
+        final String[] RomanTens = {"X", "C", "M"};
+        final String[] RomanFives = {"V", "L", "D"};
```
Comment (chicofaccin) [inline src/main/java/com/jorgepatrick/NumberConverter.java]: These arrays can be declared as statics and members of the class. So they don't need to be instantiated every time the method `setRomanDigit` is called. Also, you can use your constants instead of hardcode the value.

File: src/main/java/com/jorgepatrick/Digits.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package com.jorgepatrick;
+
+public enum Digits {
```
Comment (chicofaccin) [inline src/main/java/com/jorgepatrick/Digits.java:3]: 👏  `enum` is a powerful tool to improve the code quality! What about converting the `RomanSymbols` to an enum as well? 
**PS:** If you do that, you can do it in a separate PR.

File: src/test/java/com/jorgepatrick/NumberConverterTest.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class NumberConverterTest {
```
Comment (chicofaccin) [inline src/test/java/com/jorgepatrick/NumberConverterTest.java:11]: 🥇 Great!! 👍  forgot about what I said here -> https://github.com/JorgePatrick/RomanCalculatorGrade/pull/3

File: src/main/java/com/jorgepatrick/NumberConverter.java (hunk 2 above)
Comment (JorgePatrick) [reply src/main/java/com/jorgepatrick/NumberConverter.java:73]: I thought index variables were always i, j, k, etc, by convention

Comment (chicofaccin) [reply src/main/java/com/jorgepatrick/NumberConverter.java:73]: General is used `i` for (index), but, if you can add the meaning, it's easier to read. What `j` means?
